protos/events.proto

```diff
@@ -23,41 +23,42 @@
  syntax = "proto3";
 
  package dev.sigstore.events.v1;
-  
+
  import "google/protobuf/any.proto";
  import "google/protobuf/timestamp.proto";
- 
+
  option go_package = "github.com/sigstore/protobuf-specs/gen/pb-go/events/v1";
  option java_package = "dev.sigstore.proto.events.v1";
  option java_multiple_files = true;
- 
+ option ruby_package = "Sigstore::Events";
```

Comment:
NB: I'm not 100% confident this is the right name, but it's consistent with `envelope.proto` and others.